e2e/wdio/platform/fixtures/testData/radio-button-group.ts

```diff
@@ -0,0 +1,111 @@
+const radioButtonWithDefaultStateAndSelectionIndex = 0;
+const radioButtonInlineCozyRadioGroup = 1;
+const radioButtonWithDefaultPropertyValues = 2;
+const radioButtonWithDisabledButtonAndValidationError = 3;
+const radioButtonUsingFormGroupAndFormControl = 4;
+const radioButtonInlineWithDefaultSelection = 5;
+const radioButtonInlineCompact = 6;
+const radioButtonWithDisabledItem = 7;
+const radioButtonHaveValidationError = 8;
+const radioButtonWithNoSelectionValue = 9;
+const radioButtonWithFormGroupAndFormControl = 10;
+const radioButtonWithPreSelection = 11;
+const radioButtonWithGivenListOfStringValues = 12;
+const radioButtonWithListOfSelectItemObjects = 13;
+const radioButtonVertical = 14;
+const radioButtonVerticalQualificationForm = 15;
+const radioButtonWithSelectItemsFormGroupAndFormControl = 18;
+const radioButtonWithPreSelectionOnObjectDataPassed = 19;
+const radioButtonWithPreSelectionBasedOnControlValue = 21;
+const platformRadioButtonWithSelectiemObject = 22;
+const radioButtonWithLookupKeyAndDisplayKey = 23;
+const radioButtonWithDisabledContentAndFormcontrol = 24;
+const radioButtonWithSomeDisabledButtons = 25;
+const radioButtonCreatedFromList = 26;
+const radioButtonDisabledWithSelectedItem = 27;
+const radioButtonWithDisabledButtonsFromSelectitemObject = 28;
+const radioButtonWithDisabledButtonsDrivenForm = 29;
+const radioButtonWithIndividualButtonDisabled = 30;
+const radioButtonDisabledCreatedFromList = 31;
+const radioButtonDisabledCreatedFromSelectItem = 32;
+const radioButtonWithDisabledItemsCreatedFromSelectItem = 33;
+const resetDisabledValidationGroupButtonIndex = 21;
+const resetHaveValidationErrorIndex = 44;
+const resetListOfSelectitemObjectIndex = 51;
+const resetVerticalRadioGroupButtonIndex = 54;
+const resetQualificationFormButtonIndex = 59;
+const submitDisabledRadioButtonWithValidationErrorIndex = 20;
+const submitRadioButtonHaveValidationError = 43;
+const submitRadioButtonWithListOfSelectItemObjects = 50;
+const orientation = ['vertical', 'horizontal'];
+const monthNames = ['january', 'february', 'march', 'april'];
+const seasonsNames = ['winter', 'spring', 'summer', 'autumn'];
+const winterValue = 'winter';
+const januaryValue = 'january';
+const autumnValue = 'autumn';
+const noneValue = 'none';
+const marchValue = 'march';
+const samsungValue = 'samsung';
+const springValue = 'spring';
+const springIndex = 1;
+const marchIndex = 2;
+const autumnIndex = 3;
+const ariaChecked = 'aria-checked';
+const ngReflectIsDisabled = 'ng-reflect-is-disabled';
```

Comment:
@nikvalor why not use `export const` and remove this `export` monster?

Reply:
@Seamoo13 As I remember you said me to do in this way. What and how should we do here? Just to be on the same page